go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/satyrius/gonx
+
+go 1.17
+
+require github.com/smartystreets/goconvey v1.7.2
```

Comment:
do we have to add `goconvey` here? it's a dev dependency for tests.

Reply:
yes @satyrius it would be included in go.mod